source/web_integration/pnapi_web_integration.rst

```diff
@@ -353,4 +389,3 @@ to just this:
 
   <script type="text/javascript" src="//solutions.invocacdn.com/js/pnapi_integration-latest.min.js"></script>
 
-
```

Comment:
I am concerned about this line removal. It wasn't in the last change, and RTD can be very very particular about new lines